lmp/bb-config.sh

```diff
@@ -72,6 +72,7 @@ cat << EOFEOF >> conf/local.conf
 CONNECTIVITY_CHECK_URIS = "https://www.google.com/"
 
 ACCEPT_FSL_EULA = "1"
+LICENSE_FLAGS_ACCEPTED:rpi += "synaptics-killswitch"
```

Comment:
Maybe use append instead? Otherwise if LICENSE_FLAGS_ACCEPTED is set to a generic value, it will be replaced entirely due the :rpi override (unless it started with LICENSE_FLAGS_ACCEPTED:rpi = "foobar").

Reply:
good catch! thanks